signatures/dilithium/dilithium-shake/param/medium.yaml

```diff
@@ -0,0 +1,12 @@
+name: medium
+security level:
+  classical: 91
+  quantum: 100
+  nist category: 1
+  comment: Best known LWE attack
```

Comment:
We should add some explanation like "Authors assume their levels to understate the actual security of the schemes" or something, because in principle the categories don't fit

Reply:
Done, but below.